### PR TITLE
Fix jsdoc typings

### PR DESCRIPTION
**Pull request by ivanpopelyshev**
I am once again asking to approve a jsdoc fix.

@Zyie , @eXponenta made a few mistakes in conversion PRs.

I, @ivanpopelyshev , made two ANY's caused two messages that were bothering us from december.

### CODE REVIEW

**Comment by eXponenta** [comment]
Nice, men!
We remove added types!
Brunch should be reverted.

File: types/loader.d.ts [hunk 1]
```diff
@@ -29,22 +24,6 @@ declare namespace PIXI {
         destroy(): void;
     }
 
-    export type ICallbackID = any;
```
Comment (eXponenta) [inline types/loader.d.ts]: about this

**Comment by ivanpopelyshev** [comment]
I dont understand whats wrong there, lets discuss in slack

**Comment by ivanpopelyshev** [comment]
Yes, we have a problem with loader typings, but its not related to my changes. I'm gonna create separate issue.

**Pull request merged**